calico-cloud/image-assurance/scanners/cluster-scanner.mdx

```diff
@@ -79,6 +79,35 @@ To change default settings, modify the `imageassurance` installation resource.
 
     Set the `containerdVolumeMountPath`. Default: `/var/lib/containerd/`.
 
+### Configure exclusions for image scanning
+
+To specify which namespaces should be excluded from future scans follow the following steps.
+
+- Modify your [Image Assurance](../reference/installation/ia-api.mdx#image-assurance.operator.tigera.io/v1.ImageAssuranceSpec) installation resource to include the `exclusions.namespaces` field. List each namespace name you want to exclude.
+
+
+            ```
```

Comment:
```suggestion
            ```yaml
```

Reply:
RE formatting:

To have the code block stay in line with the list item it follows, make sure that the code block starts in the same column as the beginning of the list text. Here, it must be in line with the 'M' in 'Modify', or col 3.

If it's not meant to be part of the list item, then have the code block start on col 1.

Reply:
I'm not seeing the indentation change. Can you take another look?

Reply:
Oh I'll take a look, I commited directly last time in github, will verify this time.

Reply:
Added with 5797bc96d52370b46b1d9d291dd5acd2b26390ba lmk if that looks ok.